src/controllers/house.controllers.js

```diff
@@ -13,6 +13,36 @@ const createNewHouseController = async (houseData) => {
   }
 };
 
+export const getAll = async () => {
+  const data = await House.find({}).exec();
+
+  if (!data || !data.length) {
+    return { status: 'error', message: 'Error while fetching houses.' };
+  }
+
+  return { status: 'success', data };
+};
+
+export const getHouseList = async () => {
+  const getAllResponse = await getAll();
+
+  if (getAllResponse.status === 'error') {
+    return { status: getAllResponse.status, message: getAllResponse.message };
+  }
+
+  const { data: houses } = getAllResponse;
+
+  const data = houses.map((house) => {
```

Comment:
It's great that the structure of single house instances is flat in the data. It's going to be so much easier to work with on the front-end than with a nested structure. Would it be possible to get this kind of data right away in a query? If it was possible, this operation would probably become faster.